index.html

```diff
@@ -306,9 +234,73 @@ <h1>CONTACT US</h1>
               <a href="#"><img src="./icons/em.png" alt="image" /></a>
             </li>
           </ul>
-          <span class="email">wuyepabdul@gmail.com</span>
         </div>
-      </footer>
-    </main>
+      </div>
+    </section>
+
+    <!-- contact section -->
+    <section class="contact-section d-grid align-center padding">
+      <div>
+        <h1 class="reference"></h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <div>
+        <form
+          class="d-flex align-center"
+          action="https://formspree.io/f/xoqypwwp"
+          method="POST"
+        >
+          <input
+            class="full-width input-padding"
+            type="email"
+            name="name"
+            placeholder="Text Input"
+            minlength="30"
+            required
+          />
+          <input
+            class="full-width input-padding"
+            type="email"
+            name="email"
+            placeholder="jeremy@gmail.com"
+            required
+          />
+          <textarea
+            class="full-width input-padding"
+            name="message"
+            placeholder="Enter text here"
+            minlength="500"
+            required
+          ></textarea>
```

Comment:
For the UX (user experience) purpose, kindly make sure that the placeholder message are seen by the users. Your form doesn't display it.

Reply:
Thanks for the feedback. The Placeholder text displays  in my browser. Find attached screenshot in readme file